extensions/wikia/AdEngine/js/AdContext.js

```diff
@@ -75,9 +76,8 @@ define('ext.wikia.adEngine.adContext', [
 		}
 
 		// SourcePoint recovery integration
-		if (!context.opts.delayEngine && context.opts.sourcePointDetection && context.opts.sourcePointRecoveryUrl) {
-			context.opts.sourcePointRecovery = isUrlParamSet('sourcepointrecovery') ||
-				geo.isProperGeo(instantGlobals.wgAdDriverSourcePointRecoveryCountries);
+		if (context.opts.sourcePointProxyCSS) {
+			context.opts.sourcePointRecovery = true;
```

Comment:
Can't we get rid of this if statement and set `sourcePointRecovery` from php level?